packages/rum/package.json

```diff
@@ -32,7 +32,7 @@
     "test:unit": "npm run script runUnitTests packages/rum",
     "test:e2e:supported": "npm run script runE2eTests packages/rum/wdio.conf.js",
     "test:e2e:failsafe": "npm run script runE2eTests packages/rum/wdio-failsafe.conf.js",
-    "test:sauce": "npm run script runSauceTests packages/rum true test:unit test:e2e:supported test:e2e:failsafe",
+    "test:sauce": "npm run script runSauceTests packages/rum true build:e2e test:unit test:e2e:supported test:e2e:failsafe",
```

Comment:
Make sure we build the e2e before helps the developer experience when making changes in the test. (We were already doing the build in the other packages containing e2e btw)